getting-started-guides/python/Uninstrumented/readme.md

```diff
@@ -76,5 +77,19 @@ Set the Application Name and New Relic [Ingest - License Key](https://docs.newre
    ```bash
    ./load-generator.sh
    ```
-   
+
+5. This will only collect logs. In order to get traces and metrics you need to [make the following update to app.py](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/flask/flask.html#id1)
+    ```
+    from opentelemetry.instrumentation.flask import FlaskInstrumentor
+
+    app = Flask(__name__)
+
+    FlaskInstrumentor().instrument_app(app)
+    ```
+
+Then run the app again with:
+
+    opentelemetry-instrument --logs_exporter otlp python app.py
```

Comment:
I'll be honest I'm not sure why you now need to include `--logs_exporter otlp` in this command when we didn't before adding the metric and traces instrumentation. Empirically this was necessary for me though

Reply:
@pnvnd @reese-lee Do you know why this is necessary? Are you able to track down any OpenTelemetry Python documentation that describes why including `--logs_exporter otlp` is necessary when it is not necessary for metrics and traces?